example/sample1/federatedclusterrolebinding-placement.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Can we remove following from the yaml template? Ditto for the following yaml file.

```yaml
apiVersion: v1
kind: List
items:
```

Reply:
@gyliu513 I will update the PR shortly. Thanks for the review!